getting-started/Installation/minimal-configuration.md

```diff
@@ -0,0 +1,24 @@
+# Minimal Configuration
+
+Amber can be run from a single file for minimal configuration setups if the user prefers this for smaller applications.
+
+```crystal
+require "amber"
+
+class HelloController < Amber::Controller::Base
+  def index
+    "hello world"
+  end
+end
+
+Amber::Server.configure do |app|
+  pipeline :api do
+  end
+
+  routes :api do
+    get "/", HelloController, :index
+  end
+end
+
+Amber::Server.instance.run
```

Comment:
Isn't `Amber::Server.start` ?

Reply:
Amber::Server.run

Reply:
@eliasjpr Well, our default app template is using `.start` https://github.com/amberframework/amber/blob/master/src/amber/cli/templates/app/src/%7B%7Bname%7D%7D.cr#L3